webextension/package.json

```diff
@@ -1,4 +1,11 @@
 {
+    "name": "LanguageTool",
+    "version": "0.0.1",
+    "description": "LanguageTool - Grammar and Style Checker",
+    "main": "index.js",
+    "repository": "https://github.com/languagetool-org/languagetool-browser-addon",
+    "author": "Daniel Naber (http://www.danielnaber.de)",
+    "license": "MIT",
```

Comment:
It's actually GNU Lesser General Public License v2.1

Reply:
Ok, I changed that.